svelte-kit-scss/src/lib/helpers/issue-milestone.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  IssueMilestoneState,
+  type GithubIssueMilestone,
+  type IssueMilestone,
+} from '$lib/interfaces';
+
+export const remapIssueMilestone = (milestone: GithubIssueMilestone): IssueMilestone => {
```

Comment:
What does the `re` part of `remapIssueMilestone` mean?

This function basically validates that `milestone.state` is valid right?

Maybe we just need an `isMilestone` validater function and just pass the API return values through that?

Reply:
I will add a suggestion as a future ticket